homeassistant/components/nest/strings.json

```diff
@@ -11,6 +11,13 @@
           "code": "[%key:common::config_flow::data::access_token%]"
         }
       },
+      "pubsub": {
+        "title": "Create Pub/Sub Subscription",
```

Comment:
I think that instead of mentioning "Pub/Sub" we should instead communicate that we're going to configure their Google Cloud Project.

Title: Configure Google Cloud

Description:

```
Visit the [Cloud Console]({url}) to find your Google Cloud Project ID.
```

Reply:
Updated.